deltachat-ffi/src/dc_array.rs

```diff
@@ -1,10 +1,13 @@
+use crate::chat::ChatItem;
+use crate::constants::{DC_MSG_ID_DAYMARKER, DC_MSG_ID_MARKER1};
 use crate::location::Location;
 use crate::message::MsgId;
 
 /* * the structure behind dc_array_t */
 #[derive(Debug, Clone)]
 pub enum dc_array_t {
     MsgIds(Vec<MsgId>),
```

Comment:
I sort of expected this to be replaced by the new `Vec<ChatItem>`.  Who still uses this?

Reply:
> I sort of expected this to be replaced by the new `Vec<ChatItem>`. Who still uses this?

`get_chat_media` returns vector of `MsgId`s. Of course it can be made to return `ChatItem`s too, but I think it is good that its current type clearly shows that it can't return markers.

Reply:
Oh yes, that is indeed a very good reason!